scripts/clone-operator-repo.sh

```diff
@@ -65,8 +65,15 @@ function git_clone_checkout {
 
     pushd ${OPERATOR_BASE_DIR}
 
-    echo "Cloning repo: git clone ${git_opts} ${branch} ${repo} ${operator}-operator${ALT_CHECKOUT:+-$ALT_CHECKOUT}"
-    git clone ${git_opts} ${branch} ${repo} ${operator}-operator${ALT_CHECKOUT:+-$ALT_CHECKOUT}
+    if [ ! -d ${operator}-operator${ALT_CHECKOUT:+-$ALT_CHECKOUT} ]; then
```

Comment:
is this change necessary as the checkout should always be deleted by edpm_deploy_cleanup. I don't think this will be an issue any longer with your other Makefile changes.

Reply:
you are right!